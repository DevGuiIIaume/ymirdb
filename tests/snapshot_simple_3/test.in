set a 1
set b 2
set c 3
snapshot
list entries
del a
del b
list entries
snapshot
set a 4
set b 5
snapshot
list entries
checkout 1
list entries
del c
del b
del a
list entries
checkout 2
list entries
del c
list entries
checkout 3
list entries
del b
del a
del c
list entries
checkout 1
list entries
checkout 2
list entries
checkout 3
list entries
bye
