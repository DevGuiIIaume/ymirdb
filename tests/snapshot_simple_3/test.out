> ok

> ok

> ok

> saved as snapshot 1

> c [3]
b [2]
a [1]

> ok

> ok

> c [3]

> saved as snapshot 2

> ok

> ok

> saved as snapshot 3

> b [5]
a [4]
c [3]

> ok

> c [3]
b [2]
a [1]

> ok

> ok

> ok

> no entries

> ok

> c [3]

> ok

> no entries

> ok

> b [5]
a [4]
c [3]

> ok

> ok

> ok

> no entries

> ok

> c [3]
b [2]
a [1]

> ok

> c [3]

> ok

> b [5]
a [4]
c [3]

> bye
